app/uk/gov/hmrc/bindingtariffclassification/scheduler/DaysElapsedJob.scala

```diff
@@ -29,7 +30,7 @@ import scala.concurrent.Future
 import scala.concurrent.duration.FiniteDuration
 
 @Singleton
-class DaysElapsedJob @Inject()(appConfig: AppConfig, caseService: CaseService) extends ScheduledJob {
+class DaysElapsedJob @Inject()(appConfig: AppConfig, caseService: CaseService, bankHolidaysConnector: BankHolidaysConnector) extends ScheduledJob {
```

Comment:
double check that `appConfig.daysElapsed` reads the correct full path from application.conf